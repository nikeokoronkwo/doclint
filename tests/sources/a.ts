/**
 * @fixme {https://github.com/nikeokoronkwo/doclint} Function adds two rather than one
 */
function _addOne(a: number): number {
    a += 2
    return a
}

/**
 * @todo TA101 - Work on myself
 */
export const _takeMeAway = 0;