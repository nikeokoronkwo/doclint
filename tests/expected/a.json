[
  {
    "id": "doclint/todo",
    "message": "TA101: Work on myself"
  },
  {
    "id": "doclint/fixme",
    "message": "Function adds two rather than one",
    "hint": "See https://github.com/nikeokoronkwo/doclint for more info"
  }
]